railties/test/application/rake/dbs_test.rb

```diff
@@ -29,11 +29,9 @@ def set_database_url
       def db_create_and_drop(expected_database)
         Dir.chdir(app_path) do
           output = `bin/rails db:create`
-          assert_empty output
```

Comment:
Can we replace these `assert_empty` checks with checks for the new behavior you implemented?